### PR TITLE
New armors and weapons - up to nov 2021

### PR DESCRIPTION
**Pull request by Atemo**
* **Addressed Issue(s)**: N/A

* **Server Mode**: renewal

* **Description of Pull Request**: 

This PR adds new weapon and armor according to KRO description - up to november 2021

### CODE REVIEW

File: db/re/item_db_equip.yml [hunk 1]
```diff
@@ -116389,6 +116603,15 @@ Body:
     MagicAttack: 175
     Range: 1
     Slots: 2
+    Jobs:
```
Comment (Lemongrass3110) [inline db/re/item_db_equip.yml]: Looks wrong. Should be Summoner only?

Comment (Atemo) [reply db/re/item_db_equip.yml]: The item is https://www.divine-pride.net/database/item/26151

Comment (Lemongrass3110) [reply db/re/item_db_equip.yml]: grafik

Thank you github!

**Comment by EditorFc** [comment]
suptype
some 2hstaff wrong Suptype

**Pull request merged**